connectors/utils.py

```diff
@@ -472,3 +472,7 @@ def get_pem_format(key, max_split=-1):
     key = " ".join(key.split("\n", max_split))
     key = " ".join(key.rsplit("\n", max_split))
     return key
+
+
+class ConfigurableFieldValueError(Exception):
```

Comment:
Should we move this to `source.py` as `DataSourceConfiguration` also lives there and it feels like both belong together?

Reply:
Yeah makes sense, will do that